_posts/2021-06-08-product-management-in-open-source.md

```diff
@@ -0,0 +1,105 @@
+
+In open source software, those responsibilities and activities are a shared task of the users. By users we mean not only end users but also developers from both public and private organizations. Together, these users will have to invest time and money. The more people use the product and invest in it, the more you as a user can benefit from each other's efforts. As a user you are therefore responsible for product management. This requires commitment, knowledge and perseverance.
+
+## What is product management?
+
+[Product management](https://en.wikipedia.org/wiki/Product_management) is a collection of product development and product marketing activities within a products or services organization. The purpose of these activities is to develop a product that meets market demand on the one hand and to manage the product throughout its life cycle on the other. At a large company like Apple, thousands of people work on this every day. The end goal is to make a product or service that matches the market so well that as many as possible can be sold. The activities and responsibilities are centralized because only one organization develops the product and determines what happens to it.
+
+In open source communities there is not one single person or organization responsible for completing these activities. The big difference with commercial products is therefore that the users themselves determine what happens to the product. Of course, not every open source community is organized in the same way. Open source communities also make joint agreements about how they deal with product management. Coming to these agreements is an organic and evolutionary process. It starts informally and as the community grows, more agreements will be made and documented in the community's governance.
+
+## Product management within open source
+
+Not all possible product management activities and roles will be relevant within an open source community. Also, these activities are performed differently within various open source communities. What they have in common is that these activities are largely decentralized and the goal is to identify where there is shared interest and need - in other words, the greatest common denominator. Thus changing coalitions are formed of people who have different requirements and wishes for the product. The coalitions then organize themselves to get the resources to meet their needs.
+
+
+## Roadmaps: yes or no
+
+Eric Herman, in addition to being a codebase steward at the Foundation for Public Code, works as a board chair at [MariaDB Foundation](https://en.wikipedia.org/wiki/MariaDB). Because this community is large, the governance is managed by a separate foundation that handles a number of financial and support issues for the open source project. I asked him how product management is done within MariaDB.
```

Comment:
```suggestion
[Eric Herman](https://publiccode.net/who-we-are/team/eric-herman.html), in addition to being a codebase steward at the Foundation for Public Code, works as a board chair at [MariaDB Foundation](https://en.wikipedia.org/wiki/MariaDB). Because this community is large, the governance is managed by a separate foundation that handles a number of financial and support issues for the open source project. I asked him how product management is done within MariaDB.
```